WooCommerce/Classes/Analytics/WooAnalyticsStat.swift

```diff
@@ -1268,7 +1268,20 @@ enum WooAnalyticsStat: String {
     case backgroundUpdatesDisabled = "background_updates_disabled"
 
     // MARK: Point of Sale events
+    case pointOfSaleLoaded = "pos_loaded"
```

Comment:
This one and others that contain "_pos_" in the event itself most likely will change, as "_pos_" will be repeated otherwise. Asked here: pdfdoF-6hn#comment-7484-p2

Reply:
Sorry I didn't thought about the AFK, I could have added a link to the previous PR for context 🙇 . I agree it's something I'd like to see explicitly as well as makes much easier to reason about.